examples/gptj-qlora/config.yml

```diff
@@ -0,0 +1,57 @@
+base_model: EleutherAI/gpt-j-6b
```

Comment:
perhaps this should be moved into `gptj/config-qlora.yml` for `model/config-adapter.yml` format.